applications/virtual-fly-brain/client/src/components/StackViewerComponent.js

```diff
@@ -860,7 +874,7 @@ import React from 'react';
     createStatusText: function () {
       if (!this.state.buffer[-1]) {
         var style = {
-          font: '12px Helvetica',
+          font: '8px Helvetica',
```

Comment:
styling wise this is still way bigger than what we have in prod atm, we can bring it down again to match the label styling we have in prod. Thanks

Reply:
This one ended up being that the style object itself was not being used, PIXI had a new way of styling their textures , issue fixed in outdated view and font can be adjusted accordingly if not appropriate still.